templates/web/base/report/new/form_user_loggedin.html

```diff
@@ -18,7 +18,7 @@
 [% BLOCK form_as %]
     <label for="form_as">[% loc('Report as') %]</label>
     <select id="form_as" class="form-control js-contribute-as" name="form_as">
-        <option value="myself" [% c.user.from_body AND ( c.user.has_body_permission_to('planned_reports') || c.user.has_body_permission_to('default_to_body') ) ? '' : 'selected'  %]>[% loc('Yourself') %]</option>
+        <option value="myself" [% c.user.from_body AND ( c.user.has_body_permission_to('planned_reports') || c.user.has_body_permission_to('default_to_body') ) ? '' : 'selected'  %]>[% loc('Yourself') %] [% IF c.user.name %]([% c.user.name %])[% END %]</option>
```

Comment:
Does this need an equivalent change in the update form?

Reply:
added